db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create(
+  email: "tom@myspace.com",
+  
+)
+
+
+
+create_table "users", force: :cascade do |t|
+  t.string   "email",                  default: "", null: false
+  t.string   "encrypted_password",     default: "", null: false
+  t.string   "reset_password_token"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.